plugins/inputs/kube_inventory/README.md

```diff
@@ -55,6 +55,9 @@ See the [CONFIGURATION.md][CONFIGURATION.md] for more details.
   ## Namespace to use. Set to "" to use all namespaces.
   # namespace = "default"
 
+  ## Node name to filter to. No filtering by default.
+  # node_name = ""
```

Comment:
How do you plan on using this? Do you have a plugin for every single node you want to monitor? My concern is how does that scale?

Reply:
i use it as a daemonset and give it a node name using an env var from Kubernetes
         ```
   - name: NODE_NAME
              valueFrom:
                fieldRef:
                  apiVersion: v1
                  fieldPath: spec.nodeName
```

Reply:
So each daemonset is monitoring a single node? Is there a scenario where someone wants to pass multiple nodes?

Reply:
not that i can think of 
monitoring multiple nodes and not all nodes will require the client to use permanent node names and its a bad practice 
maybe in the future, someone would want to add some kind of prefix search but it will not be as efficient

Reply:
ok thanks!